using-timescaledb/continuous-aggregates.md

```diff
@@ -1,163 +1,220 @@
 # Continuous Aggregates
-Aggregate queries which touch large swathes of time-series data can take a long
-time to compute because the system needs to scan large amounts of data on every
-query execution. TimescaleDB continuous aggregates automatically calculate the
-results of a query in the background and materialize the results. Queries to the
-continuous aggregate view are then significantly faster as they do not need to
-touch the raw data in the hypertable, instead using the pre-computed aggregates
-in the view.
-
-Continuous aggregates are somewhat similar to PostgreSQL
-[materialized views][postgres-materialized-views], but unlike a materialized view,
-continuous aggregates do not need to be refreshed manually; the view will be refreshed
-automatically in the background as new data is added, or old data is
-modified. Additionally, it does not need to re-calculate all of the data on
-every refresh. Only new and/or invalidated data will be calculated.  Since this
-re-aggregation is automatic, it doesn’t add any maintenance burden to your
-database.
-
-Another approach would be to use triggers to update a materialized view
-immediately upon writing to the database; however, this approach causes
-significant write amplification, which slows inserts. We have designed
-continuous aggregations to run in the background and have a minimal impact on
-insertion rate by avoiding write amplification.
-
-**How it Works:**
-In general, a continuous aggregate *materialization job* takes raw data from the
-original hypertable, aggregates it, and stores intermediate state in a
-*materialization hypertable*. When you query the *continuous aggregate view*,
-the state is finalized and returned to you as needed. We also create highly
-performant triggers on the raw hypertable which determine when raw data needs to
-be re-materialized due to INSERTs, UPDATEs, or DELETEs.  The re-materialization
-will happen the next time the materialization job runs.
+
+Aggregate queries which touch large swathes of time-series data can
+take a long time to compute because the system needs to scan large
+amounts of data on every query execution. TimescaleDB continuous
+aggregates automatically calculate the results of a query in the
+background and materialize the results. Queries to the continuous
+aggregate view are then significantly faster as they do not need to
+touch the raw data in the hypertable, instead using the pre-computed
```

Comment:
Note that with real-time aggregation, this isn't strictly true anymore. We still touch raw data by default.